homeassistant/components/binary_sensor/w800rf32.py

```diff
@@ -0,0 +1,133 @@
+"""
+Support for w800rf32 binary sensors.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/binary_sensor.w800rf32/
+
+"""
+import logging
+
+import voluptuous as vol
+
+from homeassistant.components.binary_sensor import (
+    DEVICE_CLASSES_SCHEMA, PLATFORM_SCHEMA, BinarySensorDevice)
+from homeassistant.components.w800rf32 import (W800RF32_DEVICE, CONF_OFF_DELAY)
+from homeassistant.const import (CONF_DEVICE_CLASS, CONF_NAME,
+                                 ATTR_NAME, CONF_DEVICES)
+from homeassistant.core import callback
+from homeassistant.helpers import config_validation as cv
+from homeassistant.helpers import event as evt
+from homeassistant.util import dt as dt_util
+from homeassistant.helpers.dispatcher import (async_dispatcher_connect)
+
+_LOGGER = logging.getLogger(__name__)
+
+DEPENDENCIES = ['w800rf32']
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
+    vol.Optional(CONF_DEVICES, default={}): {
+        cv.string: vol.Schema({
+            vol.Optional(CONF_NAME): cv.string,
+            vol.Optional(CONF_DEVICE_CLASS): DEVICE_CLASSES_SCHEMA,
+            vol.Optional(CONF_OFF_DELAY):
+            vol.Any(cv.time_period, cv.positive_timedelta)
```

Comment:
Should we only accept positive timedelta periods?
```suggestion
            vol.All(cv.time_period, cv.positive_timedelta)
```